docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -91,6 +91,33 @@ When I wait until element located by `name(welcome-image)` disappears
 ----
 
 
+=== Wait for element with specified state using polling interval
+
+Waits for an element with the specified state to be found using the specified timeout with polling interval.
+
+[source,gherkin]
+----
+When I wait `$duration` with `$pollingDuration` polling until element located by `$locator` becomes $state
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.6.0)_:
+[source,gherkin]
+----
+When I wait `$duration` with `$pollingDuration` polling until element located `$locator` becomes $state
+----
+
+* `$duration` - Total duration to wait in the {iso-date-format-link} format.
+* `$pollingDuration` - The duration to wait between search retries in the {iso-date-format-link} format.
+* `$locator` - <<_locator>>.
+* `$state` - xref:parameters:state.adoc[State].
```

Comment:
can we inject the partial instead of link here?

Reply:
> can we inject the partial instead of link here?

It won't be look good without rewriting of state.adoc (we can't include certain header)
![image](https://github.com/vividus-framework/vividus/assets/58571912/f9a4d26b-5706-4e95-94ac-69c36cfa524a)